src/logic/calculate.test.js

```diff
@@ -0,0 +1,6 @@
+import calculate from './calculate';
+
+it('should return an object', () => {
+  const calculateTest = calculate({ total: 0, next: null, operation: null }, 'AC');
+  expect(calculateTest).toStrictEqual({ total: 0, next: null, operation: null });
+});
```

Comment:
- [ ] Kindly make sure that all the functionalities & scenarios of the `calculate.js` (numbers, operations: +, - , * , / , % )  should be covered using Jest. And add a good description for every test.

Reply:
Hi @omarsalem7 , please I don't understand what this review says. Please can you explain?

Reply:
Thank you for taking your time to reach out @omarsalem7